### PR TITLE
Flatzebra: Updated and added new recipe for 0.1.6

### PR DESCRIPTION
**Pull request by lich**

### CODE REVIEW

File: dev-games/flatzebra/flatzebra-0.1.6.recipe [hunk 1]
```diff
@@ -0,0 +1,62 @@
+DESCRIPTION="A generic game engine for 2D double-buffering animation"
+SUMMARY="Generic game engine"
+HOMEPAGE="http://sarrazip.com/dev/burgerspace.html"
+LICENSE="GNU GPL v2"
+COPYRIGHT="2009 Pierre Sarrazin."
```
Comment (humdingerb) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: move COPYRIGHT under HOMEPAGE

File: dev-games/flatzebra/flatzebra-0.1.6.recipe [hunk 2]
```diff
@@ -0,0 +1,62 @@
+DESCRIPTION="A generic game engine for 2D double-buffering animation"
+SUMMARY="Generic game engine"
```
Comment (humdingerb) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: move SUMMARY to the top

File: dev-games/flatzebra/flatzebra-0.1.6.recipe [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+LICENSE="GNU GPL v2"
+COPYRIGHT="2009 Pierre Sarrazin."
+SOURCE_URI="http://perso.b2b2c.ca/sarrazip/dev/flatzebra-$portVersion.tar.gz"
+CHECKSUM_SHA256="4e7bb0a77136ec3b81e0f73c1d08e828d38ef011095d5ce7068a94f3bb21d67a"
+REVISION="2"
+SOURCE_DIR="flatzebra-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86_64 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+
```
Comment (humdingerb) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: remove one blank line

File: dev-games/flatzebra/flatzebra-0.1.6.recipe [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+PROVIDES="
+	flatzebra = $portVersion
+	lib:libflatzebra_0.1$secondaryArchSuffix = $portVersion
+	"
+
```
Comment (humdingerb) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: no blank line here

File: dev-games/flatzebra/flatzebra-0.1.6.recipe [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_image$secondaryArchSuffix
+	devel:libsdl_mixer$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	"
+
```
Comment (humdingerb) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: no blank lie here

File: dev-games/flatzebra/flatzebra-0.1.6.recipe (hunk 3 above)
Comment (lich) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: Sorry, but according to the wiki guidelines there should be blank line after ARCHITCTURES segment.
https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines

Comment (humdingerb) [inline dev-games/flatzebra/flatzebra-0.1.6.recipe]: Yes, but there were two, weren't there? No matter... it's OK now.

**Pull request merged**